BUILD

```diff
@@ -181,6 +181,13 @@ cc_library(
     deps = [":protobuf_lite"],
 )
 
+cc_library(
+    name = "protobuf_headers",
+    hdrs = glob(["src/**/*.h"]),
+    includes = ["src/"],
+    visibility = ["//visibility:public"],
```

Comment:
@damienmg what is the best way to restrict visibility to tensorflow only here, such that it works for both the cases, i.e., TF compiled by itself, and TF as a submodule? I was thinking something like:

    visibility = ["@org_tensorflow//:internal"]

but not sure if that would work in both cases. Also, what is the succinct description of the Bazel problem I can add here? Is there a tracking issue that I can simply link?